vnext/JSI.Desktop.UnitTests/JSI.Desktop.UnitTests.vcxproj

```diff
@@ -0,0 +1,145 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <!-- We need to figure out how we want to consume the Google Test Nuget for
+    Win32. For now, we disable the Win32. -->
+    <!-- <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration> -->
```

Comment:
>uration> --> [](start = 19, length = 12)

remove commented out lines. File a separate work item if this still needs to be done

Reply:
Filed #2702; let's keep the commented out configuration for now though,

Reply:
There is no reason to keep it. If it doesn't work, it just adds unneeded complexity and makes files harder to read. Please remove.

---
In reply to: [298773698](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r298773698) [](ancestors = 298773698)